.github/workflows/nightly_repeated_unittest.yml

```diff
@@ -0,0 +1,46 @@
+name: Amplify Nightly Unit Test
```

Comment:
suggestion: renaming to Amplify Nightly Repeated Unit Test Run

Reply:
Changed